packages/vaadin-lumo-styles/badge.js

```diff
@@ -86,24 +86,20 @@ const badge = css`
 
   /* Icon */
 
-  [theme~='badge'] vaadin-icon,
-  [theme~='badge'] iron-icon {
+  [theme~='badge'] vaadin-icon {
     margin: -0.25em 0;
-    --iron-icon-width: 1.5em;
-    --iron-icon-height: 1.5em;
+    width: 1.5em;
+    height: 1.5em;
```

Comment:
There are no custom properties in `vaadin-icon`, so using CSS properties instead.

Reply:
nit: The fixed values will override the defaults of `var(--lumo-icon-size-m)` that also happen to be 1.5em by default. But if `--lumo-icon-size-m` was adjusted, this would no longer scale...not sure if it's an issue

Reply:
Good point, thanks. I will revert this change for now, we can address it later if needed.